buildSrc/src/main/java/org/elasticsearch/gradle/Util.java

```diff
@@ -16,30 +16,24 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.elasticsearch.gradle.vagrant
 
-import org.gradle.api.tasks.Input
+package org.elasticsearch.gradle;
 
-/**
- * Runs bats over vagrant. Pretty much like running it using Exec but with a
- * nicer output formatter.
- */
-public class BatsOverVagrantTask extends VagrantCommandTask {
-
-    @Input
-    Object remoteCommand
+import org.gradle.api.GradleException;
 
-    BatsOverVagrantTask() {
-        command = 'ssh'
-    }
-
-    void setRemoteCommand(Object remoteCommand) {
-        this.remoteCommand = Objects.requireNonNull(remoteCommand)
-        setArgs((Iterable<?>) ['--command', remoteCommand])
-    }
+public class Util {
 
-    @Override
-    protected OutputStream createLoggerOutputStream() {
-        return new TapLoggerOutputStream(logger, getProgressLoggerFactory().newOperation(boxName).setDescription(boxName));
+    public static boolean getBooleanProperty(String property, boolean defaultValue) {
+        String propertyValue = System.getProperty(property);
+        if (propertyValue == null) {
+            return defaultValue;
+        }
+        if ("true".equals(propertyValue)) {
```

Comment:
Do we want to enforce case here?

Reply:
Yes. This is a copy from code we have in Elasticsearch to be non-lenient in our boolean parsing.